src/sagemaker/estimator.py

```diff
@@ -2209,7 +2209,14 @@ def _validate_and_set_debugger_configs(self):
         ):
             self.debugger_hook_config = DebuggerHookConfig(s3_output_path=self.output_path)
         elif not self.debugger_hook_config:
-            self.debugger_hook_config = None
+            self.debugger_hook_config = False
```

Comment:
Why did this change from `None` to `False`?

Reply:
The line above intends to disable the `self.debugger_hook_config`, setting it to `False` is a stronger indication than `None`